packages/components/inputs/search-select-input/src/search-select-input.js

```diff
@@ -0,0 +1,210 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { SafeHTMLElement } from '@commercetools-uikit/utils';
+import AsyncSelectInput from '@commercetools-uikit/async-select-input';
+import {
+  CustomSelectInputOption,
+  SearchIconDropdownIndicator,
+} from '@commercetools-uikit/select-utils';
+import messages from './messages';
+
+const SearchSelectInput = (props) => {
+  const intl = useIntl();
+  const noOptionsMessage =
+    props.noOptionsMessage ||
+    (() => intl.formatMessage(messages.noOptionsMessage));
+  const loadingMessage =
+    props.loadingMessage || intl.formatMessage(messages.loadingOptionsMessage);
+
+  return (
+    <AsyncSelectInput
+      {...props}
+      components={{
+        // eslint-disable-next-line react/display-name
+        Option: (optionInnerProps) => (
+          <CustomSelectInputOption
+            optionType={props.optionType}
+            optionInnerProps={optionInnerProps}
+          />
+        ),
```

Comment:
I sometimes wonder if this should live outside. So `const components = { Option: OptionComponent }` and `const OptionComponent = props =>` cause this may cause quite some re-renders.

Reply:
added the requested changes